pkg-manager/plugin-commands-installation/test/dedupe.ts

```diff
@@ -108,6 +111,88 @@ describe('pnpm dedupe', () => {
     expect(fs.existsSync('package.json')).toBeTruthy()
     expect(fs.existsSync('output.json')).toBeFalsy()
   })
+
+  describe('respects .npmrc options', () => {
+    test('uses store-dir .npmrc option', async () => {
```

Comment:
Note that this test was already passing before this PR. The `rcOptionTypes` defined by `install` are present in the `config/config/src/index.ts` file.

https://github.com/pnpm/pnpm/blob/693944b66994f50f5f1a940e10e0f0dd0c98dae2/config/config/src/index.ts#L43-L50

That's merged with the command-specific `rcOptionTypes` here, which taught `pnpm dedupe` to understand them.

https://github.com/pnpm/pnpm/blob/c597f72ec3d3076e0ee84afb9d12c67d4666e334/config/config/src/index.ts#L202

This PR only affects CLI flags as a result.

Reply:
I'm not sure we need these tests. If we have these tests here should we add them to all commands? They are expensive e2e tests.

Reply:
They are pretty expensive. I would be convinced with the argument that they're more trouble than they're worth.

I'll replace these with a unit test that just asserts `cliOptionTypes()` has some fields that we're expecting.